### ISSUE TITLE
Questions about training data

### ISSUE THREAD

**Issue by hrz2000** [autonomousvision/plant#2]
![image](https://user-images.githubusercontent.com/79900945/201456080-7513f193-17b7-4468-8e6c-6d47418fa39b.png)
1. Whether the files in the red box are the files that generate the training set of plantT

2. What is the meaning of the "routes_training.xml" below the red box? It seems that the number of routes is only 50, but it has more waypoints per route. Is it a collection of files in the red box above?

3. Are lav and longest6 used only as validation sets? Because I noticed that the number of their routes is very small

4. the data_agent_boxes.py encapsulated autopilot experts conducted simulation tests and found that the results were very poor. The simulation would hit the wall soon after the start of the simulation, while the transferuser experts were normal. Have you noticed this problem?

Maybe my question will be a little silly. I'm a beginner and I look forward to your early reply.

**Comment by RenzKa** [comment]
Hi,
thanks for your interest in our work.

1. Yes we used the routes in the red box to generate our training set. You can check out the datagen.sh file to get the combinations of scenario files and route files (route file is the path after "eval.route_rel" and scenario file is the path after "eval.scenarios_rel"). You can also check out the transfuser repo for more information about the dataset.

2. I removed them as they are not relevant to this work.

3. We only used the longest 6 routes for online evaluation in our paper. For more information on this benchmark (e.g. number of routes etc. check out the transfuser paper and repo). I included LAV as another benchmark in case someone wants to run our model on the LAV routes. 

4. Can you tell me with which command you started your simulation test? Then I can look into it.

Best,
Katrin

**Comment by hrz2000** [comment]
Thank you for your prompt reply. I just noticed that planT's warehouse has been updated. The newly added code is very useful to us. 
For my fourth question above, I investigated again, and found that I ignored the DATAGEN parameter when changing the agent, which led to the instantiation of the self.agent_instance is actually a planT without loading parameters. 
Thank you for your warm and detailed answers again!
